likecoin/js/admin-settings/src/pages/SponsorLikecoinPage.js

```diff
@@ -108,18 +120,35 @@ function SponsorLikecoinPage() {
       <h3>{__('Preserve Content', 'likecoin')}</h3>
       <p>{localizedDecentralizedRegistry}</p>
 
+      <hr />
       <h2>{__('What is LikeCoin', 'likecoin')}</h2>
       <p>{localizedLikeCoinIntroduction}</p>
 
       <h2>{__('Why Web3', 'likecoin')}</h2>
       <p>{localizedWeb3Introduction}</p>
 
+      <hr />
+      <h2>{__('Subscribe to our Newsletter', 'likecoin')}</h2>
+      <p>{localizedSubscribe}</p>
+      <iframe
+          src="https://newsletter.like.co/embed"
+          width="100%"
+          height="320"
```

Comment:
Why is this height 320 not 150 like above?

Reply:
The full size widget is 320 height, with logo and title. I think it's too large and confusing to see in getting start page, so I reduced the height and let its responsiveness kick in to hide the title and logo

Reply:
So why not reduce it here too?

Reply:
no idea... do you think the smaller one looks better here? yes then we can reduce it here also

Reply:
reduced to 150